src/generate/config.ts

```diff
@@ -24,6 +24,8 @@ export interface OptionalConfig {
   columnOptions: ColumnOptions;
   schemaJSDoc: boolean;
   unprefixedSchema: string | null;
+  types?: Record<string, Record<string, string>>;
```

Comment:
aren't all of the types here already optional? so I don't think you need to make these two optional?

Reply:
Yeah, I was avoiding setting the defaults, but we can do that.